script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt

```diff
@@ -55,10 +55,10 @@ I feel the warmth of her concern as she tells me that we may have just been misu
 "But I can't be sure of that anymore.
 He just recently passed away. Because his death was so sudden, those seven years of misunderstanding are now over, unresolved."
 ...Beyond that.
-That is if there ever truly was a misunderstanding.
+That is, if there ever truly was a misunderstanding.
 <Page17>
-"So, now that your father is gone.
-You're to come home to your parents' house, they said."
+"So, now that your father is gone now, huh.
+You're to return to your family home, they said."
```

Comment:
First line has spurious now, and the pair of lines reads a bit awkward overall. Suggest
> "I see, so your father has passed away.
> And so they said you were to return to the family home?"